backend/src/libs/packages/server-application/libs/helpers/check-is-white-route/check-is-white-route.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { type HTTPMethod } from '#libs/packages/http/http.js';
+import { WHITE_ROUTES } from '#libs/packages/server-application/server-application.js';
+
+const checkIsWhiteRoute = ({
+  path,
+  method,
+}: {
```

Comment:
Create a type for this before checkIsWhiteRoute declaration. Let's split types and code functionality)